CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [0.0.11]
```

Comment:
You need to update `package.json` because that is the authoritative location of the version.